README.md

```diff
@@ -1,12 +1,29 @@
 
-[![CircleCI](https://circleci.com/gh/woocommerce/woocommerce-ios.svg?style=svg)](https://circleci.com/gh/woocommerce/woocommerce-ios)
-[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
 
-# WooCommerce for iOS
+<h1 align="center"><img src="https://woocommerce.com/wp-content/themes/woo/images/logo-woocommerce.svg" width="300"><br>for iOS</h1>
```

Comment:
nit: maybe we can download the image and load the static image here? just in case it might save some network requests 😛

Reply:
I see that there's actually an asset added to the PR (`logo-woocommerce.svg`), maybe we can show it here?